src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -1394,10 +1394,6 @@ def _validate_final_average_model(self, cleanup_snapshots_pkl_file=False):
         if not self.ddp_silent_mode:
             # Adding values to sg_logger
             # looping over last titles which corresponds to validation (and average model) metrics.
```

Comment:
please remove the documentation line as well

Reply:
done